packages/router/router.ts

```diff
@@ -2256,8 +2256,14 @@ export function unstable_createStaticHandler(
       };
     }
 
-    // Create a GET request for the loaders
-    let loaderRequest = new Request(request.url, { signal: request.signal });
+    // Create a request for the loaders
+    let loaderRequest = new Request(request.url, {
+      body: null,
+      headers: request.headers,
+      method: request.method,
+      redirect: request.redirect,
+      signal: request.signal,
+    });
```

Comment:
Same as Remix was doing in 1.7.6: https://github.com/remix-run/remix/blob/remix%401.7.6/packages/remix-server-runtime/server.ts#L418

Reply:
For the moment we are not persisting `method:'POST'` as Remix did since we think that should be changed in Remix